sdk/textanalytics/ai-text-analytics/package.json

```
@@ -86,7 +86,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-client": "^1.0.0",
     "@azure/core-rest-pipeline": "^1.1.0",
-    "@azure/core-lro": "^2.3.0",
+    "@azure/core-lro": "^2.3.0-beta.1",
```

This should be an exact pin.

```suggestion
    "@azure/core-lro": "2.3.0-beta.1",
```

---

Committing this at @deyaaeldeen's request.